backend/contributions/ai_review/views.py

```
@@ -108,17 +115,27 @@ def filter_exclude_username(self, queryset, name, value):
         return queryset
 
     def filter_assigned_to(self, queryset, name, value):
```

Ruff (0.15.15)

[warning] Unused method argument: `name`

(ARG002)